templates/android/build.gradle.twig

```diff
@@ -2,15 +2,15 @@ apply plugin: 'io.github.gradle-nexus.publish-plugin'
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.31"
-    version '{{ sdk.version }}'
+    ext.kotlin_version = "1.5.31"
+    version "{{ sdk.version }}"
```

Comment:
Are double quotes and single quotes treated similarly ?

Reply:
Yep they are the same except allow for interpolation. I changed for consistency but I see there's other I missed anyway